src/main/java/com/sda/games/checkers/model/Game.java

```diff
@@ -1,71 +1,155 @@
 package com.sda.games.checkers.model;
 
+import lombok.*;
+
+import java.util.ArrayList;
 import java.util.List;
 import java.util.Scanner;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
 public class Game {
-    static String whitePlayerName;
-    static String blackPlayerName;
 
-    private Player[] players;
+    private List<Player> players;
     private Board board;
-    private Player currentTurn;
+    private Player currentTurnPlayer;
     private GameStatus status;
     private List<Move> movesPlayed;
+    private Scanner scanner = new Scanner(System.in);
+
+    public void newGame() {
+        System.out.println("Enter White Player name:");
+        Player whitePlayer = new Player(1, Player.whitePlayerName = scanner.nextLine(), true);
+        System.out.println("Enter Black Player name:");
+        Player blackPlayer = new Player(2, Player.blackPlayerName = scanner.nextLine(), false);
 
-    public Game(Player[] players) {
-        this.players = players;
+
+        initialize(whitePlayer, blackPlayer);
+        getBoard().printBoard();
     }
 
     public void initialize(Player player1, Player player2) {
-        players[0] = player1;
-        players[1] = player2;
+        this.players = new ArrayList<>();
+        players.add(player1);
+        players.add(player2);
 
+        this.board = new Board();
         board.resetBoard();
 
-        if (player1.getWhite()) {
-            this.currentTurn = player1;
+        if (player1.isWhite()) {
```

Comment:
this jest tu niepotrzebny, jezeli masz tylko jedna zmienna nazwana currentTurnPlayer to kompilator bedzie wiedziec ze to jest pole klasy :) 
BTW mozna szybciej krocej ` currentTurnPlayer = player1.isWhite() ? player1 : player2;`